### ISSUE TITLE
Flattr icon

### ISSUE THREAD

**Issue by ThomDietrich** [heiseonline/shariff#174]
The icon shown for flattr (`fa-money`) may be descriptive... but it's not the flattr icon. I feel a lot of users would recognize the flattr icon instantly, the money bill is not telling the user anything and is mostly avoided on my page.
![flattr-logo-og](https://cloud.githubusercontent.com/assets/2870104/16359830/c61a148a-3b44-11e6-8fee-da0c3bc32f03.png)

**Comment by craiq** [comment]
This won't happen until the symbol is available in font awesome.
You can modify it yourself by creating a new font with e.g. fontello.com
When I see it right the metric settings of font awesome are "Units per em" = 1792, "Ascent" = 1536 and "Descent" = -256
Remember the code of the symbol, add the font to you page and add the following css:
`.shariff .fa-money:before { content: "\[symbol code]" !important}`
you might also need to adjust `.shariff .fa-money { font-size: [...]}`
Another problem is, that the symbol is not easy to identify with just one color and this size.

**Comment by core23** [comment]
Flattr doesn't allow to use their icon :(

**Comment by ThomDietrich** [comment]
"doesn't allow"? What makes you say that?
Flattr icons are available all over the place, e.g. http://www.entypo.com
Entypo and other font packages contain flattr icons, fontawesome does not.

Why is Shariff using the one font that doesn't cover the 15 services it deals with? o.O
Why not use another or it's own with only the ones needed?

**Comment by compeak** [comment]
We know about the issues with font awesome and are currently thinking about moving to vector icons (see #154).

**Comment by stefahn** [comment]
I second @ThomDietrich: Shariff should use the original icons.

Another reason for moving to new (own) icons: The currently "shipped" Google plus icon (g+) is outdated. The current one has a capital G+, see https://developers.google.com/+/branding-guidelines#1-button-branding-guidelines

**Comment by ThomDietrich** [comment]
Interesting, I didn't know that till now. And here we thought google+ is dead ;)
However somebody was quicker than you. The "g+" is now a "G+". Check it out https://heiseonline.github.io/shariff
That aside, independent vector icons are a good idea!

**Comment by stefahn** [comment]
Yes, Google+ has never really come to life. Maybe it will eventually.

I saw the new G+ icon too, but [on my install](https://marketingunited.org) I still have the old g+ icon. I cloned the master only a week ago. What could be wrong?
